libraries/botbuilder/src/main/java/com/microsoft/bot/builder/core/BotAdapter.java

```diff
@@ -69,7 +69,7 @@ public BotAdapter Use(Middleware middleware) {
     /// an array of <see cref="ResourceResponse"/> objects containing the IDs that
     /// the receiving channel assigned to the activities.</remarks>
     /// <seealso cref="ITurnContext.OnSendActivities(SendActivitiesHandler)"/>
-    public abstract CompletableFuture<ResourceResponse[]> SendActivities(TurnContext context, Activity[] activities) throws InterruptedException;
+    public abstract CompletableFuture<ResourceResponse[]> SendActivities(TurnContext context, Activity[] activities) throws InterruptedException, ServiceKeyAlreadyRegisteredException;
```

Comment:
heads up - using a custom exception does reflect the current state of the C# however that is not best practice and we will be updating the c#

Reply:
Waiting with anticipation..